nautobot/core/templates/inc/nav_menu.html

```diff
@@ -56,18 +56,11 @@
                                                                 {% endif %}
                                                             {% endfor %}
                                                         </div>
-                                                        {% comment %}
-                                                            Use 'url xxx as variable' so that an invalid
-                                                            link doesn't throw a NoReverseMatch exception.
-                                                        {% endcomment %}
-                                                        {% url item_link as item_url %}
-                                                        {% if item_url %}
-                                                            <a href="{{ item_url }}"
+                                                        {% if item_link %}
+                                                            <a href="{{ item_link }}"
                                                                data-item-weight="{{ item_details.weight }}">
                                                                 {{ item_details.name }}
                                                             </a>
-                                                        {% else %}
```

Comment:
Don't we still need an `else` case here to render something if the link is no good?

Reply:
I made the item.name the original `"ERROR: Invalid link!"` message in the `except NoReverseMatch` block. It will automatically render `"ERROR: Invalid link!"` if the link is no good.

Reply:
Is the `if item_link` check no longer correct/necessary then?

Reply:
Oof you are right